src/facade/dragonfly_listener.h

```diff
@@ -45,6 +45,7 @@ class Listener : public util::ListenerInterface {
 
   // Returns thread-local dynamic memory usage by TLS.
   static size_t TLSUsedMemoryThreadLocal();
+  static uint64_t TLSMaxClientsReachedCount();
```

Comment:
This name is slightly confusing.
When reading it, I thought this would be "the maximum number of concurrently connected users", but in fact it's how many times we refused a client connection.
How about `RefusedConnectionMaxClientsCount()` or something along these lines?

Reply:
sorry, it was totally copy paste typo